dashboard/test/controllers/data_docs_controller_test.rb

```diff
@@ -47,4 +47,31 @@ class DataDocsControllerTest < ActionController::TestCase
     get :show, params: {key: 'unknown_key'}
     assert_response :not_found
   end
+
```

Comment:
Could you add tests that only levelbuilder can access the edit action? It would be similar to the new action:
```
  # new page is levelbuilder only
  test_user_gets_response_for :new, user: nil, response: :redirect
  test_user_gets_response_for :new, user: :student, response: :forbidden
  test_user_gets_response_for :new, user: :teacher, response: :forbidden
  test_user_gets_response_for :new, user: :levelbuilder, response: :success
```

Reply:
On it